Sources/OutlineView/OutlineView.swift

```diff
@@ -103,6 +103,7 @@ where Data.Element: Identifiable {
     ///     element in `data`. An `NSTableCellView` subclass is preferred.
     ///     The `NSView` should return the correct `fittingSize`
     ///     as it is used to determine the height of the cell.
+    @available(macOS 11.0, *)
```

Comment:
Neither is this one or you "need" to provide an alternative for lower versions. But in its current form it works on 10.15

Reply:
On macOS 10.15 the separator seems to be laid out differently. If I use the `AdjustableSeparatorRowView` the separator  is not positioned between the cells instead it seems to be positioned ~1 px above the highlighted rect for a selected row. 

So for now I decided to only support it on macOS 11 and up. But if someone else wants to use a separator, it can always be added later. 🙂